### PR TITLE
HYRAX-357

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Updated hyrax-metadata-updates with new OPeNDAP URL format.

Addresses [HYRAX-357:node.js development for Cumulus](https://bugs.earthdata.nasa.gov/browse/HYRAX-357?filter=-1)

From [this PR](https://github.com/nasa/cumulus/pull/2116)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

**Comment by vskorole** [comment]
Should not to remove getEntryTitle function in the _index.js_ because it is used in _test-integration-index.js_ and _test-search-failures.js_

File: tasks/hyrax-metadata-updates/index.js [hunk 1]
```diff
@@ -101,21 +101,23 @@ async function getEntryTitle(config, metadata, isUmmG) {
...
+  const conceptId = get(result, '[0].id');
+  shortName = get(result, '[0].short_name');
+  versionId = get(result, '[0].version_id');
+  if (conceptId === undefined) {
+    throw new RecordDoesNotExist(`Unable to query parent collection using short name ${shortName} and version ${versionId}`);
   }
 
-  return encodeURIComponent(datasetId);
+  return (config.addShortnameAndVersionIdToConceptId !== undefined
+    && config.addShortnameAndVersionIdToConceptId === true)
```
Comment (nemreid) [inline tasks/hyrax-metadata-updates/index.js:119]: Given that this conditional checks that `config.addShortnameAndVersionIdToConceptId` is both defined AND true, can't we just strict-check for true?

File: tasks/hyrax-metadata-updates/tests/test-integration-index.js [hunk 2]
```diff
@@ -204,7 +204,7 @@ test.serial('Test updating ECHO10 metadata file in S3', async (t) => {
     });
```
Comment (nemreid) [inline tasks/hyrax-metadata-updates/tests/test-integration-index.js:204]: The `beforeEach` in this test file still references the `entry title` concept.

**Review by nemreid — CHANGES_REQUESTED** [review]
I have a few small things I'd like to see addressed, then I can approve this.

**Review by laurenfrederick — COMMENTED** [review]
@nemreid addressed comments. Thanks for the review!

**Review by nemreid — APPROVED** [review]
Great 💯

**Pull request merged**